lib/ssh_scan/api_db.rb

```diff
@@ -0,0 +1,61 @@
+require 'sqlite3'
+
+module SSHScan
+  class APIDatabaseHelper
+    def initialize(database_name)
+      if File.exist?(database_name)
+        @db = ::SQLite3::Database.open(database_name)			
+      else
+        @db = ::SQLite3::Database.new(database_name)
+        self.create_schema
+      end
+    end
+
+    def create_schema
+      @db.execute <<-SQL
+        create table api_schema (
+          worker_id varchar(100),
+          uuid varchar(100),
+          result json
```

Comment:
We may need to take a second pass on the schema for the DB, but as is, it works.  I would like to be able to query elements within the scan results eventually, but I think it could be a future improvement and not affect user/worker interfaces.